card/mod/05_standard/set/all/rich_html/menu.rb

```diff
@@ -2,23 +2,33 @@
   view :menu, :denial=>:blank, :tags=>:unknown_ok do |args|
     return _render_template_closer if args[:menu_hack] == :template_closer
     return '' if card.unknown?
-    content_tag :div, :class=>'menu-slot' do
-      (_optional_render(:horizontal_menu, args, :hide) || _render_menu_link(args)) +
+    wrap_with :div, :class=>'menu-slot' do
+      [
+        _optional_render(:horizontal_menu, args, :hide),
+        _render_menu_link(args),
         _render_modal_slot(args)
+      ]
     end
   end
 
   view :menu_link do |args|
     path_opts = {:slot => {:home_view=>args[:home_view]}}
     path_opts[:is_main] = true if main?
-    content_tag :div, :class=>'btn-group pull-right slotter card-slot card-menu vertical-card-menu' do
-      view_link(glyphicon(args[:menu_icon]), :vertical_menu, :path_opts=>path_opts).html_safe
+    css_class = if show_view?(:horizontal_menu,args.merge(:default_visibility=>:hide, :optional=>true))
+                  'visible-xs'
+                else
+                  'show-on-hover visible-xs'
```

Comment:
as long as we dont have detection of touch devices we can use this to make the menu visible on small screens